### PR TITLE
CableLabs Adjustments

### PR DESCRIPTION
**Pull request by trevormiles**
## Description

This PR brings over some tweaks from CableLabs so that it is easier for me to copy the JS between the two repos. The less differences, the less time I have to take making sure I maintain differences between the two.

### CODE REVIEW

File: src/scene/index.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ function onMouseDown(event) {
   // Handle Event: "Clicked Nothing" (deselect/hide selections)
   if (!intersects || !intersects[0]) {
     clearHighlightedPoint();
-    const selection = window.document.getElementsByClassName('location visible')[0];
+    const selection = window.document.getElementsByClassName('globe-location-card visible')[0];
```
Comment (schwindy) [inline src/scene/index.js:97]: Hmm, if you needed to change this to be getting elements with these two classes:
1. `globe-location-card`
2. `visible`

Then I think the Location Marker "Auto-Select" functionality will also need this change. In earth.js, a similar line to this one exists where it is grabbing by `location visible` classes 🐒

File: src/scene/index.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ export function isHovering() {
 export function render() {
   const freeze = !hover || drag || !GLOBE_HOVER_FREEZE_ENABLED;
   if (freeze) {
-    target.x += 0.00075;
+    target.x += 0.001185;
```
Comment (schwindy) [inline src/scene/index.js:76]: We could make this a constant so it's more easily configurable (GLOBE_ROTATION_SPEED or something) 🐒

File: src/scene/index.js (hunk 1 above)
Comment (schwindy) [reply src/scene/index.js:97]: PS: I am blind, you did this already 🐒 nvm

**Review by schwindy — APPROVED** [review]
👍 Looks good to me, thanks! 🐒

File: src/scene/index.js (hunk 2 above)
Comment (trevormiles) [reply src/scene/index.js]: Done 👍

**Pull request merged**